README.md

```diff
@@ -40,6 +40,16 @@ Requires the following:
 -   Node.js (latest LTS recommended)
 -   Bower 1.8.8+
 
+### Git Pre-commit installation
+
+Our git [pre-commit](https://pre-commit.com/#install) [hooks](/.pre-commit-config.yaml) allow you to run flake8 and black upon `git commit` and save you from having to run these linters manually.
+
+- change into the base directory
+- `pip install pre-commit`
```

Comment:
should this be installed via pip or conda?

Reply:
@eamahanna  dunno. It can be installed just for this project, or for your entire machine. The way I wrote it allows you to install it based on your current python environment